tests/models/bert/test_modeling_tf_bert.py

```diff
@@ -590,7 +591,7 @@ def prepare_config_and_inputs_for_common(self):
 
 
 @require_tf
-class TFBertModelTest(TFModelTesterMixin, TFCoreModelTesterMixin, unittest.TestCase):
+class TFBertModelTest(TFModelTesterMixin, TFCoreModelTesterMixin, PipelineTesterMixin, unittest.TestCase):
```

Comment:
Not `TFPipelineTesterMixin` here: the framework could be inferred within `PipelineTesterMixin` itself